### PR TITLE
Update dependencies, java version, gradle version, gradle plugins

### PR DESCRIPTION
**Pull request by mzcydev**

### CODE REVIEW

**Comment by Koboo** [comment]
Hey @whynotmax 
Thank you very much for keeping this up-to-date! :) 
Currently I'm more focused on other projects, so I didn't really had time for TemplateStom. 

I added one comment, which you could resolve. After that I'm happy to merge your PR into the project. <3

File: server-app/src/main/java/eu/koboo/minestom/server/ProjectVariables.java [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 public class ProjectVariables {
 
-  public static final String NAME = "PROJECT_NAME";
-  public static final String VERSION = "PROJECT_VERSION";
-  public static final String MINESTOM_VERSION = "DEP_MINESTOM_VERSION";
+  public static final String NAME = "TemplateStom";
+  public static final String VERSION = "1.0.2";
+  public static final String MINESTOM_VERSION = "8f46913486";
```
Comment (Koboo) [inline server-app/src/main/java/eu/koboo/minestom/server/ProjectVariables.java]: We don't want to replace it everytime we release a new version, so I implemented the gradle plugin blossom, which automatically replaces the Strings here with their respective gradle values. 
(See here: https://github.com/whynotmax/TemplateStom/blob/main/server-app/build.gradle#L47)

Please revert these changes :)

Comment (mzcydev) [reply server-app/src/main/java/eu/koboo/minestom/server/ProjectVariables.java]: Yup, my bad, didn't think it would do this in java. Will revert

**Comment by mzcydev** [comment]
@Koboo review, please <3

**Comment by Koboo** [comment]
Looks good to me. Thank you for your contribution! <3
